src/test/scala/org/broadinstitute/dsde/firecloud/EntityServiceSpec.scala

```diff
@@ -214,10 +217,72 @@ class EntityServiceSpec extends BaseServiceSpec with BeforeAndAfterEach {
     }
   }
 
+  "EntityService.listJobs" - {
+    "should concatenate results from cWDS and Import Service" in {
+      val importServiceResponse = List(
+        ImportServiceListResponse("jobId1", "status1", "filetype1", None),
+        ImportServiceListResponse("jobId2", "status2", "filetype2", None)
+      )
```

Comment:
It weirds me out that these two list have identical jobIds.  Not clear how/if that will impact the test

Reply:
copy/paste error, fixed!